documentation/ios_tutorial.md

```diff
@@ -80,6 +80,23 @@ latch.startProximityUnlock()
 latch.stopProximityUnlock()
 ```
 
+## Sync
+
```

Comment:
@duff-latch @Ian-latch We should add something here about what a sync actually is? And what situations one might want to do one?